xbmc/threads/SystemClock.h

```diff
@@ -69,4 +71,9 @@ namespace XbmcThreads
     inline unsigned int GetInitialTimeoutValue(void) const { return totalWaitTime; }
     inline unsigned int GetStartTime(void) const { return startTime; }
   };
+
+  inline void ThreadSleep(unsigned int millis)
+  {
+    std::this_thread::sleep_for(std::chrono::microseconds(millis));
```

Comment:
if I am not mistaken std:chrono caused major issues on Android
@peak3d @koying do you know more?

Reply:
That was maybe an old NDK?

Reply:
@MartijnKaijser https://github.com/xbmc/xbmc/pull/13378